samples/subsys/fs/spi_sd/src/main.c

```diff
@@ -0,0 +1,307 @@
+/*
+ * Copyright (c) 2019 Tavish Naruka <tavishnaruka@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* Sample which uses the filesystem API and SDHC driver */
+
+#include <zephyr.h>
+#include <device.h>
+#include <disk/disk_access.h>
+#include <logging/log.h>
+#include <fs/fs.h>
+#include <ff.h>
+
+LOG_MODULE_REGISTER(main);
+
+/*-------------------------------------------------------------------------
+ * Defines
+ *-------------------------------------------------------------------------
+ */
+#define TEST_FILE_SIZE 1024
+#define KILO 1000
```

Comment:
This results in an unusual transfer size, neither 10^6 nor 2^20.  It could be more consistent with normal testing if the multiplier were 1024, so the total transfer size is 1 MiBy.

Reply:
OK to set blocks of 1024  x 1024 bytes